src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -1021,6 +1025,19 @@ def gen_cert(host, addr):
                       tls_config:
                         ca_file: root_cert.pem
 
+                  - job_name: 'smb'
+                    honor_labels: true
+                    scheme: https
+                    tls_config:
+                      ca_file: root_cert.pem
+                    http_sd_configs:
+                    - url: https://[::1]:8765/sd/prometheus/sd-config?service=smb
```

Comment:
make check failure here is only bcz it must be `http` here instead
```suggestion
                    - url: http://[::1]:8765/sd/prometheus/sd-config?service=smb
```